### ISSUE TITLE
tests_views.py:569-572: Create arch for the Search tests....

### ISSUE THREAD

**Issue by 0pdd** [fidals/stroyprombeton#651]
The puzzle `60m-1d4cf566` from #60m has to be resolved: 

https://github.com/fidals/stroyprombeton/blob/7db5d298ead54df6abefca44d96c2be9c35d8d08/stroyprombeton/tests/tests_views.py#L569-L572

The puzzle was created by duker33 on 15-May-19. 

  

If you have any technical questions, don't ask me, submit new tickets instead. The task will be \"done\" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

**Comment by 0pdd** [comment]
@artemiyds2 @duker33 please pay attention to this new issue.

**Comment by duker33** [comment]
depends on #652
